project_stage/README.md

```diff
@@ -0,0 +1,17 @@
+# Project Stage
+
+This module adds the stages on project like stages on task.
+
+![Task Form](static/description/project_stages.png?raw=true)
+
+## Configuration
+
+The project types are editable under Project / Settings / Project Stages.
+
+![Project Type List](static/description/project_stage_tree.png?raw=true)
+
+![Project Type List](static/description/project_stage_form.png?raw=true)
+
+Contributors
```

Comment:
Codacy found an issue: [[heading-style] Headings should use atx](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6165869)